packages/database/lib/schema.ts

```diff
@@ -275,3 +297,31 @@ export const candles = pgTable('candles', {
 }, table => ({
   pk: primaryKey(table.marketAcct, table.candleDuration, table.timestamp)
 }));
+
+export const comments = pgTable('comments', {
+  // Need this as we reference this for response and nesting
+  commentId: bigint('comment_id', {mode: 'bigint'}).notNull(),
+  // Generated when comment is created
+  commentorAcct: pubkey('commentor_acct').notNull(),
+  proposalAcct: pubkey('proposal_acct').notNull(),
+  // This will be the body content of the comment
+  content: text('content').notNull(),
+  // Use only if its a responding comment in a chain, we should constrain this so
+  // it references only commentIds which have respondingCommentId with NULL
+  respondingCommentId: bigint('responding_comment_id', {mode: 'bigint'}).references(() => comments.commentId),
```

Comment:
Not sure how to do what I want to do here. It seems like it's a function or check, of which check doesn't exist in drizzle.........